pyne/alara.py

```diff
@@ -843,3 +852,57 @@ def _get_subvoxel_array(mesh, cell_mats):
 
     return subvoxel_array
 
+def _convert_unit_to_s(dc):
+    """
+    This function return a float number represent a time in unit of s.
+    Parameters
+    ----------
+    dc : string. Contain a num and an unit. 
+
+    return: a float number 
+    """
+    # get num and unit
+    [num, unit] = dc.split()    
+    if unit in ('s', 'second'):
+        return float(num)
+    if unit in ('m', 'minute'):
+        return float(num) * 60
+    if unit in ('h', 'hour'):
+        return float(num) * 60 * 60
+    if unit in ('d', 'day'):
+        return float(num) * 60 * 60 * 24
+    if unit in ('w', 'week'):
+        return float(num) * 60 * 60 * 24 * 7
+    if unit in ('y', 'year'):
+        return float(num) * 60 * 60 * 24 * 7 * 52 
+    if unit in ('c', 'century'):
+        return float(num) * 60 * 60 * 24 * 7 * 52 * 100
```

Comment:
First, these should all be `elif` after the first.

Reply:
Second, there is a better way to do this.  First put all of your conversion factors in a global (module-level) dict.  Then, you can look up the value from this dict using either indexing or get(). I would write something like:

```python
_TO_SEC = {
  'second': 1.0,
  ...
  'century': 60 * 60 * 24 * 7 * 52 * 100.0,
}
# add abbreviations
_TO_SEC.update({k[0]: v for k, v in _TO_SEC.items()})

def _convert_unit_to_s(dc):
    num, units = dc
    conv = _TO_SEC.get(units, None)
    if conv is None:
        raise ValueError('Invalid unit: {0}'.format(unit))
    return float(num) * conv
```